galley/pkg/server/server.go

```diff
@@ -128,6 +130,12 @@ func newServer(a *Args, p patchTable) (*Server, error) {
 			if err := p.verifyResourceTypesPresence(k); err != nil {
 				return nil, err
 			}
+		} else {
+			found, err := p.findSupportedResources(k)
```

Comment:
Nit: test for this case in server_test.go

Reply:
done